{
  "cveId": "CVE-2025-59399",
  "eventName": "New CVE Received",
  "cveChangeId": "76D2A584-6FD8-44AF-8E6F-2726CF4D9B5B",
  "sourceIdentifier": "cve@mitre.org",
  "created": "2025-09-15T19:15:36.307",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "libocpp before 0.28.0 allows a denial of service (EVerest crash) because a secondary exception is thrown during error message generation."
    },
    {
      "action": "Added",
      "type": "CVSS V3.1",
      "newValue": "AV:A/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:L"
    },
    {
      "action": "Added",
      "type": "CWE",
      "newValue": "CWE-460"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/EVerest/libocpp/commit/0b84d7f9fb3c338d470770f220a7b7f21db78878"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/EVerest/libocpp/compare/v0.27.1...v0.28.0"
    }
  ]
}